### DISCUSSION TITLE
Baseline Time in OpenSAFELY-TPP

### DISCUSSION THREAD

**Discussion by annaschultze** [opensafely/documentation#401] [General]
Baseline time, or some minimum amount of time before start of FU during which comorbidity information can be recorded, is defined in different ways in different databases. As I understand it, there are two variables that can be used to determine what we normally call 'baseline time' in TPP: 

- `registered_with_one_practice_between` AND
- `with_complete_gp_consultation_history_between`

There is some explanatory text for the latter in the docs (screenshot below), but I'm not sure I understand why these two concepts would be expected to differ. I also wasn't sure if there was a good reason for requiring a certain amount of minimum practice registration time to ensure accurate comorbidity recording, if medical records are transferred automatically between GP EHR vendors (if this is the case?) 

![image](https://user-images.githubusercontent.com/59482746/132835429-f209fdd2-dd08-411b-b748-616410d504ae.png)

**Comment by wjchulme** [comment]
> I also wasn't sure if there was a good reason for requiring a certain amount of minimum practice registration time to ensure accurate comorbidity recording, if medical records are transferred automatically between GP EHR vendors

This is a good point, and one @HelenCEBM has thought about a bit before. Certainly a year seems like too much, and maybe even a month if record transfer is automatic and complete. 

But there is another reason to include a "burn-in" period after registration, which is that people often only register with a new practice when they need access to healthcare services. If I move house, I might only get round to registering with my local GP a year later when I have a reason to visit. If this is common enough, then healthcare activity is biased upwards for registered-time closer to the registration date. This matters less when you're looking back for historical clinical information, than when you're looking forward for prescriptions, referrals, etc.  So whether to account for it is study specific. Or maybe variable specific. 

A way to check this would be to look at genuine GP-patient interactions (excluding address changes, etc) as a function of time since registration, to see if it's higher at the start.
